packages/stacks/package.json

```diff
@@ -25,10 +25,12 @@
     "@leather.io/utils": "workspace:*",
     "@noble/hashes": "1.5.0",
     "@scure/bip32": "1.5.0",
-    "@stacks/encryption": "6.16.1",
-    "@stacks/network": "6.13.0",
+    "@stacks/common": "7.0.2",
+    "@stacks/encryption": "7.0.2",
+    "@stacks/network": "7.0.2",
     "@stacks/stacks-blockchain-api-types": "7.8.2",
-    "@stacks/transactions": "6.17.0",
+    "@stacks/transactions": "7.0.2",
+    "@stacks/transactions-v6": "npm:@stacks/transactions@6.17.0",
```

Comment:
@kyranjamie why are both of these installed here? Should just be `7.0.2`, correct?

Reply:
for a while I had both to deal with some transitional stuff, but yeah we should remove this